dask_ml/model_selection/methods.py

```diff
@@ -549,7 +519,6 @@ def create_cv_results(
                 n_splits,
                 n_candidates,
                 splits=True,
```

Comment:
any reason why

Reply:
I think this is just a bool for a private store function that just reminds the store function to create different keys for different splits